src/win/tty.c

```diff
@@ -1611,6 +1627,13 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
     uv_buf_t buf = bufs[i];
     unsigned int j;
 
+  if (vterm == SUPPORTED) {
+    utf16_buf_used = MultiByteToWideChar(CP_UTF8, 0, buf.base, buf.len, NULL, 0);
+    MultiByteToWideChar(CP_UTF8, 0, buf.base, buf.len, utf16_buf, utf16_buf_used);
```

Comment:
also, why is the conversion needed only if ANSI color codes are supported? can you perhaps add a comment?